MDANSE_GUI/Src/MDANSE_GUI/Tabs/Models/TrajectoryModel.py

```diff
@@ -142,6 +181,7 @@ def removeRow(self, row: int, parent: QModelIndex | None = None) -> bool:
         retcode = super().removeRow(row, parent)
         instance = self._trajectory_instances.pop(node_number)
         filename = self._trajectory_paths.pop(node_number)
+        _ = self._trajectory_status.pop(node_number)
```

Comment:
```suggestion
        self._trajectory_status.pop(node_number)
```
Don't need to return, although a perhaps more canonical way would be:

```suggestion
        del self._trajectory_status[node_number]
```